code/modules/antagonists/revolution/revolution.dm

```diff
@@ -9,6 +9,7 @@
 	antag_moodlet = /datum/mood_event/revolution
 	var/hud_type = "rev"
 	var/datum/team/revolution/rev_team
+	ghost_text = "I can't give up now, the heads must pay!"
```

Comment:
low effort

Reply:
"There is no ethical ghosting under capitalism."

Reply:
Do you hear the people ghost? No, you fucking don't!

Reply:
REVS ARENT COMMUNIST

Reply:
"Boss makes a dollar, you make a dime, so don't fucking ghost on company time."